<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Emjoy</class>
 <widget class="QWidget" name="Emjoy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/ccf.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>10</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/dqv.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>10</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/dqx.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/dqw.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/dra.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/dqy.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>10</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/dqz.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>10</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drb.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>45</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drj.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>16</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>45</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/dri.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>45</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drg.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>13</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>45</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drh.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>45</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/dre.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drc.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>45</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drd.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>45</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drf.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_17">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>115</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drv.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>28</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_18">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>115</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drz.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>32</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_19">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>115</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/dru.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>27</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_20">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drq.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>23</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_21">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/dro.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_22">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>80</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drp.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>22</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_23">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drm.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>19</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_24">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>115</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drw.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>29</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_25">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drk.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>17</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_26">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>80</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drr.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>24</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_27">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>115</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/dry.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>31</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_28">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>115</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drt.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>26</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_29">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>80</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drl.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>18</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_30">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>80</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drn.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_31">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>115</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drx.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_32">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>115</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(biaoqing/drs.png);
color:rgb(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>25</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>160</height>
    </rect>
   </property>
  </widget>
  <zorder>textBrowser</zorder>
  <zorder>pushButton_9</zorder>
  <zorder>pushButton_11</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_26</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_10</zorder>
  <zorder>pushButton_13</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_27</zorder>
  <zorder>pushButton_30</zorder>
  <zorder>pushButton_31</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>pushButton_22</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_21</zorder>
  <zorder>pushButton_23</zorder>
  <zorder>pushButton_18</zorder>
  <zorder>pushButton_14</zorder>
  <zorder>pushButton_28</zorder>
  <zorder>pushButton_16</zorder>
  <zorder>pushButton_12</zorder>
  <zorder>pushButton_25</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_15</zorder>
  <zorder>pushButton_17</zorder>
  <zorder>pushButton_20</zorder>
  <zorder>pushButton_24</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_32</zorder>
  <zorder>pushButton_19</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_29</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
